### PR TITLE
Added DeckTape conversion into builder

### PR DESCRIPTION
**Pull request by darkdrgn2k**

### CODE REVIEW

**Comment by Shrinks99** [comment]
Can the PDF be placed into: output/module-x/**handouts**/module-x-deck.pdf instead?

Also while a `worksheet` folder exists this should also be renamed to `handouts` in order to encompass all material that is related to the presentation to give out to students.

Other than that, looking good!

File: install-dependencies.sh [hunk 1]
```diff
@@ -9,6 +9,5 @@ npm install -g gitbook-cli
 # For generating course website with Jekyll
 gem install bundler
 
-# For converting remark presentations to pdf
-
+# For converting remark presentations to PDF
```
Comment (benhylau) [inline install-dependencies.sh]: But now some `pdf` are lower and some are `PDF` are upper in comments

File: build.sh [hunk 2]
```diff
@@ -51,4 +54,24 @@ for mod in module-*; do
         echo "Generating presentation from $book to $out"
         gitbook build "$book" "$out"
     fi
+
+    # Generate presentation DeckTape pdf and Remark content
```
Comment (benhylau) [inline build.sh]: `Remark content` here

File: build.sh [hunk 3]
```diff
@@ -51,4 +54,24 @@ for mod in module-*; do
...
+    doc="$mod-presentation.html"
+    cd $mod
+    if [ -f $doc ]; then
+
+        # Generate PDF
+        out="../output/$mod-presentation.pdf"
+        echo "Generating lesson plan from $doc to $out"
+        decktape remark "$doc" "$out" --chrome-arg=--allow-file-access-from-files
+
+        # Copy remark Content
```
Comment (benhylau) [inline build.sh]: `remark Content` here??

File: build.sh (hunk 2 above)
Comment (benhylau) [inline build.sh]:
```suggestion
    # Copy remark presentation and generate pdf copy with decktape
```

File: build.sh [hunk 4]
```diff
@@ -51,4 +54,24 @@ for mod in module-*; do
         echo "Generating presentation from $book to $out"
         gitbook build "$book" "$out"
     fi
+
+    # Generate presentation DeckTape pdf and Remark content
+    doc="$mod-presentation.html"
+    cd $mod
+    if [ -f $doc ]; then
+
```
Comment (benhylau) [inline build.sh]:
```suggestion
```

File: build.sh [hunk 5]
```diff
@@ -51,4 +54,24 @@ for mod in module-*; do
...
+        # Generate PDF
+        out="../output/$mod-presentation.pdf"
+        echo "Generating lesson plan from $doc to $out"
```
Comment (benhylau) [inline build.sh]:
```suggestion
        echo "Generating presentation from $doc to $out"
```

File: build.sh (hunk 3 above)
Comment (benhylau) [inline build.sh]:
```suggestion
        # Copy remark presentation
```

File: build.sh [hunk 6]
```diff
@@ -51,4 +54,24 @@ for mod in module-*; do
...
+        decktape remark "$doc" "$out" --chrome-arg=--allow-file-access-from-files
+
+        # Copy remark Content
+        cp -r "slide-images/" "../output/$mod/"
+        cp $doc "../output/$mod/"
```
Comment (benhylau) [inline build.sh]:
```suggestion
        cp "$doc" "../output/$mod/"
```

File: build.sh (hunk 3 above)
Comment (benhylau) [reply build.sh]: Move the copy up, and add:

```
echo "Copying presentation from $doc to ../output/"
```

File: build.sh [hunk 7]
```diff
@@ -51,4 +54,24 @@ for mod in module-*; do
...
+     fi
+     cd $base_dir
+
 done
+
+# Copy slide files for Remark
```
Comment (benhylau) [inline build.sh]:
```suggestion
# Copy slide files for remark
```

File: install-dependencies.sh (hunk 1 above)
Comment (darkdrgn2k) [reply install-dependencies.sh]: In this file they where all CAPS, and by ALL i mean the other ONE.

File: build.sh (hunk 3 above)
Comment (darkdrgn2k) [reply build.sh]: remark Content or Remark content

File: install-dependencies.sh (hunk 1 above)
Comment (Shrinks99) [reply install-dependencies.sh]: In the dependencies file we have referred to it as .pdf

File: build.sh (hunk 3 above)
Comment (Shrinks99) [reply build.sh]: Remark content

**Pull request merged**